### ISSUE TITLE
The link to ImageMagick-7.0.10-37-portable-Q16-x64.zip is broken

### ISSUE THREAD

**Issue by testworksau** [ImageMagick/ImageMagick#2809]
### Prerequisites

- [x] I have written a descriptive issue title
- [x] I have verified that I am using the latest version of ImageMagick
- [x] I have searched [open](https://github.com/ImageMagick/ImageMagick/issues) and [closed](https://github.com/ImageMagick/ImageMagick/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### Description

[The link](https://imagemagick.org/download/binaries/ImageMagick-7.0.10-37-portable-Q16-x64.zip) to ImageMagick-7.0.10-37-portable-Q16-x64.zip is broken in https://imagemagick.org/script/download.php

I came across this issue while looking for a binary due to [the Chocolatey package](https://chocolatey.org/packages/imagemagick.tool) failing to work for the past 24 days (not sure if this team maintains that package too).

### Steps to Reproduce

1. Navigate to https://imagemagick.org/script/download.php
2. Attempt to download ImageMagick-7.0.10-37-portable-Q16-x64.zip

### System Configuration

- ImageMagick version: 7.0.10-37-portable-Q16-x64
- Environment (Operating system, version and so on): N/A
- Additional information: N/A

**Comment by urban-warrior** [comment]
Try now.

**Comment by testworksau** [comment]
Hi @urban-warrior 👋 

I can confirm that the latest version `7.0.10-39` can be downloaded successfully. 

Version `7.0.10-37` is still missing, as is [7.0.10-38](https://imagemagick.org/download/binaries/ImageMagick-7.0.10-38-portable-Q16-x64.zip) which is what the Chocolatey package is now attempting to install.

I suspect the ftp mirror they are using needs to be updated as the latest version available there is `7.0.10-29` (06-Sep-2020).

```
ERROR: The remote file either doesn't exist, is unauthorized, or is forbidden for url 'https://ftp.icm.edu.pl/pub/graphics/ImageMagick/binaries/ImageMagick-7.0.10-38-portable-Q16-x64.zip'. Exception calling "GetResponse" with "0" argument(s): "The remote server returned an error: (404) Not Found."
```

**Comment by testworksau** [comment]
It looks like the more recent chocolatey builds are working again - I guess this can be closed.
